central/rbac/service/service_impl.go

```diff
@@ -117,6 +117,7 @@ func (s *serviceImpl) ListRoleBindings(ctx context.Context, rawQuery *v1.RawQuer
 
 // GetSubject returns the subject with the input ID (the unique subject name).
 func (s *serviceImpl) GetSubject(ctx context.Context, request *v1.ResourceByID) (*v1.GetSubjectResponse, error) {
+	// TODO(ROX-27174): Remove this endpoint because used function always returns "nil".
```

Comment:
Should the removal be announced (CHANGELOG entry) ? This can be a follow-up.

Reply:
I'll remove this comment and let it be handled within the ticket. If we decide to remove or fix this API.